package.json

```diff
@@ -51,5 +52,8 @@
     "ts-node": "^10.9.1",
     "typescript": "^4.9.3",
     "web3": "^1.8.1"
+  },
+  "dependencies": {
+    "graphql": "^16.6.0"
```

Comment:
Just leaving a comment to double check if `graphql` is a prod dependency or dev one.